### DISCUSSION TITLE
GitHub Student Developer Pack for NIOS Students

### DISCUSSION THREAD

**Discussion by Samar029** [community/community#65648] [GitHub Education]
### Hi everyone,

I am one of the National Institute of Open Schooling Noida India, and i need to be re-verify my educational detail for the GithHub Student Developer Pack previously it is verified by my ID Card but now it requires a mail id with @nios.ac.in domain. NIOS never provide email id to student with this domain then how can i re-verify my educational details  and re-activate my student developer pack.

**Comment by Shubham-C12** [comment]
Same bro im also finding the solution to this matter.
